plugins/catalog-backend-module-msgraph/README.md

```diff
@@ -106,53 +105,21 @@ yarn add --cwd packages/backend @backstage/plugin-catalog-backend-module-msgraph
  ): Promise<Router> {
    const builder = await CatalogBuilder.create(env);
 
-+  // The target parameter below needs to match one of the providers' target
-+  // value specified in your app-config (see above).
 +  builder.addEntityProvider(
 +    MicrosoftGraphOrgEntityProvider.fromConfig(env.config, {
-+      id: 'production',
-+      target: 'https://graph.microsoft.com/v1.0',
 +      logger: env.logger,
 +      schedule: env.scheduler.createScheduledTaskRunner({
-+        frequency: Duration.fromObject({ minutes: 5 }),
-+        timeout: Duration.fromObject({ minutes: 3 }),
```

Comment:
Luxon can be removed from the import section in this example since `Duration` is not used anymore :)

Reply:
yes! Seems like I missed that place. I will take care